test/executor.test.js

```diff
@@ -53,11 +53,12 @@ describe('executor', function() {
   });
 
   describe('when booting', function() {
-    it('should set the booting status', function(done) {
+    it('should set the `booting` flag during execution', function(done) {
       expect(app.booting).to.be.undefined();
-      boot.execute(app, dummyInstructions, function(err) {
+      boot.execute(app, simpleAppInstructions(), function(err) {
         expect(err).to.be.undefined();
-        expect(app.booting).to.be.false();
+        expect(process.bootingFlagSet).to.be.ok();
+        expect(app.booting).to.not.be.ok();
```

Comment:
That was by accident actually. I looked up `ok()` and `...not.be.ok()` and `true()`/`false()` makes more sense but I forgot to change it back. I will submit another PR to change it back before we forget about it.